src/snapshotter.test.js

```diff
@@ -8,6 +8,8 @@ import logger from './logger';
 jest.mock('fs');
 jest.mock('jimp');
 
+const onComplete = () => {};
```

Comment:
Maybe it would be worth having a test to check that the callback is triggered?

Reply:
done